doc/user_guide/source/HowToUseNomad.rst

```diff
@@ -251,9 +251,18 @@ Algorithmic parameters
 ``DIRECTION_TYPE``
 """"""""""""""""""
 
-The type of direction ``DIRECTION_TYPE`` for *Mads* *Poll* step must be selected in the set ``{ORTHO 2N, N+1 UNI, SINGLE, DOUBLE}``. The option ``ORTHO 2N`` corresponds to the original *Ortho Mads* algorithm [AbAuDeLe09]_ with :math:`2n` directions.
-With option ``N+1 UNI`` *Mads* *Poll* step uses :math:`n+1` uniformly distributed directions. Options ``SINGLE`` and ``DOUBLE`` produces one or two directions for *Mads* *Poll* step respectively.
+This parameter defines the type of directions for *Mads* *Poll* step. The possible arguments are:
 
+.. csv-table:: Toto
+   :widths: 5,20
+
+   ``ORTHO 2N``, "OrthoMADS, 2n. This corresponds to the original *Ortho Mads* algorithm [AbAuDeLe09]_ with :math:`2n` directions."
+   ``ORTHO N+1 NEG``, "OrthoMADS, n+1, with ((n+1)th dir = negative sum of the first n dirs)."
```

Comment:
Mettre une référence vers le papier Ortho n+1

@article{AuIaLeDTr2014,
  Author        = {C. Audet and A. Ianni and S. {Le~Digabel} and C. Tribes},
  Title         = {{Reducing the Number of Function Evaluations in Mesh Adaptive Direct Search Algorithms}},
  Journal       = {SIAM Journal on Optimization},
  Volume        = {24},
  Number        = {2},
  Pages         = {621--642},
  Url           = {https://dx.doi.org/10.1137/120895056},
  Doi           = {10.1137/120895056},
  Year          = {2014},
  Date-Added    = {2014-01-16},
  Date-Modified = {2014-12-03}
} % ex AuIaLeDTr2012